quickwit-indexing/src/models/indexed_split.rs

```diff
@@ -43,16 +45,18 @@ pub struct IndexedSplit {
     // invalid.
     pub docs_size_in_bytes: u64,
 
-    /// Instant of the instant of the first document in the split.
+    /// Instant of reception of the first document in the indexer.
+    ///
+    /// This is mostly useful to understand part of the time to search.
+    /// Note however, the document may have been waiting for a long time in the source
```

Comment:
We could actually measure that time if we wanted to. Kafka and Kinesis, at least, provide the arrival time of a message in the queue.